app/models/responses/PropertyAnnualSubmission.scala

```diff
@@ -291,8 +289,7 @@ object PropertyAnnualSubmission {
     val electricChargePointAllowanceLens = GenLens[UkOtherAllowances](_.electricChargePointAllowance)
 
     // Focuses
-    val focusFromRequestOnToannualInvestmentAllowanceLens =
-      ukOtherPropertyLens.andThen(ukOtherAllowancesLens).andThen(annualInvestmentAllowanceLens)
+    ukOtherPropertyLens.andThen(ukOtherAllowancesLens).andThen(annualInvestmentAllowanceLens)
```

Comment:
"val focusFromRequestOnToannualInvestmentAllowanceLens =" deleted?

Reply:
if it is not used, the assigned line ```ukOtherPropertyLens.andThen(ukOtherAllowancesLens).andThen(annualInvestmentAllowanceLens``` should be deleted as well.

Reply:
Scala fix plugin removes only variable its funny, removed the entire line now. Thanks for pointing it.